### ISSUE TITLE
Accessibility layers should be based on the data in the geojson rather than a fixed list.

### ISSUE THREAD

**Issue by doorleyr** [CityScope/CS_cityscopeJS#15]
Currently, the Accessibility layer always shows accessibility to Food, Education, Groceries and Nightlife. However, the relevant POIs will be different for different projects. For example, the corktown accessibility data on cityIO now also has accessibility to housing and employment (which update when new housing and employment blocks are added).

https://cityio.media.mit.edu/api/table/corktown/access

The front-end should check the first feature in the accessibility geojson to see which pois are accounted for and build the accessibility layers accordingly.

**Comment by RELNO** [comment]
what is the fastest way to get the list of these pois? fetching the first feature and parsing it's properties ?

**Comment by doorleyr** [comment]
That's what I would suggest.
